### PR TITLE
Update App.java

### PR DESCRIPTION
**Pull request by itjraymond**
doing some kaban board testing.

### CODE REVIEW

File: src/main/java/ca/jent/App.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
  * This Java source file was generated by the Gradle 'init' task.
+ * Adding a comment
```
Comment (itjraymond) [inline src/main/java/ca/jent/App.java]: Please clarify.

Comment (itjraymond) [reply src/main/java/ca/jent/App.java]: You got it!

Comment (itjraymond) [reply src/main/java/ca/jent/App.java]: Done

**Review by jocelynraymond — APPROVED** [review]
Good job!

**Pull request merged**